cli/cli/kurtosis_config/resolved_config/kurtosis_cluster_config_test.go

```diff
@@ -252,19 +265,113 @@ func TestNewKurtosisClusterConfigShouldEnableDefaultLogsSink(t *testing.T) {
 		KubernetesClusterName:  &kubernetesClusterName,
 		StorageClass:           &kubernetesStorageClass,
 		EnclaveSizeInMegabytes: &kubernetesEnclaveSizeInMB,
 		EngineNodeName:         &kubernetesEngineNodeName,
 	}
-	kurtosisClusterConfigOverrides := v5.KurtosisClusterConfigV5{
+	kurtosisClusterConfigOverrides := v6.KurtosisClusterConfigV6{
 		Type:                        &kubernetesType,
 		Config:                      &kubernetesFullConfig,
 		LogsAggregator:              nil,
+		LogsCollector:               nil,
 		GrafanaLokiConfig:           nil,
 		ShouldEnableDefaultLogsSink: &ShouldEnableDefaultLogsSink,
 	}
 	_, err := NewKurtosisClusterConfigFromOverrides("test", &kurtosisClusterConfigOverrides)
 	require.NoError(t, err)
 }
+
+func TestNewKurtosisClusterConfigLogsCollectorNoConfig(t *testing.T) {
+	kubernetesType := KurtosisClusterType_Kubernetes.String()
+	kubernetesClusterName := "some-name"
+	kubernetesStorageClass := "some-storage-class"
+	kubernetesEnclaveSizeInMB := uint(5)
+	kubernetesEngineNodeName := "some-node-name"
+	kubernetesFullConfig := v6.KubernetesClusterConfigV6{
+		KubernetesClusterName:  &kubernetesClusterName,
+		StorageClass:           &kubernetesStorageClass,
+		EnclaveSizeInMegabytes: &kubernetesEnclaveSizeInMB,
+		EngineNodeName:         &kubernetesEngineNodeName,
+	}
+	kurtosisClusterConfigOverrides := v6.KurtosisClusterConfigV6{
+		Type:                        &kubernetesType,
+		Config:                      &kubernetesFullConfig,
+		LogsAggregator:              nil,
+		LogsCollector:               nil,
+		GrafanaLokiConfig:           nil,
+		ShouldEnableDefaultLogsSink: nil,
+	}
+	actualKurtosisClusterConfig, err := NewKurtosisClusterConfigFromOverrides("test", &kurtosisClusterConfigOverrides)
+	require.NoError(t, err)
+	require.Nil(t, actualKurtosisClusterConfig.logsCollector.Filters)
+	require.Nil(t, actualKurtosisClusterConfig.logsCollector.Parsers)
+}
+
+func TestNewKurtosisClusterConfigLogsCollectorFullConfig(t *testing.T) {
+	kubernetesType := KurtosisClusterType_Kubernetes.String()
+	kubernetesClusterName := "some-name"
+	kubernetesStorageClass := "some-storage-class"
+	kubernetesEnclaveSizeInMB := uint(5)
+	kubernetesEngineNodeName := "some-node-name"
+	kubernetesFullConfig := v6.KubernetesClusterConfigV6{
+		KubernetesClusterName:  &kubernetesClusterName,
+		StorageClass:           &kubernetesStorageClass,
+		EnclaveSizeInMegabytes: &kubernetesEnclaveSizeInMB,
+		EngineNodeName:         &kubernetesEngineNodeName,
+	}
+	kurtosisClusterConfigOverrides := v6.KurtosisClusterConfigV6{
+		Type:   &kubernetesType,
+		Config: &kubernetesFullConfig,
+		LogsAggregator: &v6.LogsAggregatorConfigV6{
+			Sinks: map[string]map[string]interface{}{
+				"elasticsearch": {
+					"type": "elasticsearch",
+				},
+			},
+		},
+		LogsCollector: &v6.LogsCollectorConfigV6{
+			Filters: []logs_collector.Filter{
+				{
+					Name:  "grep",
+					Match: "*",
+					Params: []logs_collector.FilterParam{
```

Comment:
Same question as above: maybe it's fine that we're reimplmenting the logs collector config spec. Might be easier though if you just pass an arbitrary struct in here and Kurtosis will serialize it as-is to the logs collector

Reply:
addressed above